mzLib/FlashLFQ/IsoTracker/SearchingTarget.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace FlashLFQ.IsoTracker
+{
+    public class SearchingTarget
+    {
```

Comment:
Can you name this class "SearchTarget" or "IsobaricSearchTarget" and/or add summary comments?